.zenodo.json

```diff
@@ -97,6 +97,11 @@
       "name": "Avants, Brian",
       "orcid": "0000-0002-4212-3362"
     },
+    {
```

Comment:
This is duplication from line 12.

Reply:
Duplicate removed.

Reply:
```bash
fgrep '"name"' .zenodo.json | sort | uniq -c | egrep -v '^ *1 +'
```
gives
```plaintext
      2       "name": "Johnson, Hans",
      2       "name": "McCormick, Matthew",
      2       "name": "Straing, Marius"
      2       "name": "Tustison, Nick",
      2       "name": "Zuki\u0107, D\u017eenan",
```

Reply:
With caaea931fb82b9e20bddeaa9932a348896de9288 I am getting two duplicates:
```
      2       "name": "Straing, Marius"
      2       "name": "Tustison, Nick",
```

Reply:
And Straing might be a misspelling of [Staring](https://scholar.google.com/citations?user=pKFkfq4AAAAJ&hl=en)